src/platforms/python/guides/django/index.mdx

```diff
@@ -28,20 +30,7 @@ If you experience memory leaks in your channels consumers while using the SDK, y
 
 Unfortunately the SDK is not able to do so by itself, as [Channels is missing some hooks for instrumentation](https://github.com/django/channels/issues/1348).
 
-## Behavior
-
-- All exceptions leading to an Internal Server Error are reported.
-
-- Request data is attached to all events: **HTTP method, URL, headers, form data, JSON payloads**. Sentry excludes raw bodies and multipart file uploads. Sentry also excludes personally identifiable information (such as user ids, usernames, cookies, authorization headers, IP addresses) unless you set `send_default_pii` to `True`.
-
-- Each request has a separate scope. Changes to the scope within a view, for example setting a tag, will only apply to events sent as part of the request being handled.
-
-- If you use `django.contrib.auth` and have set `send_default_pii=True` in your call to `init`, user data (current user id, email address, username) is attached to the event.
-
-- The Sentry Python SDK will attach SQL queries as breadcrumbs.
 
-- Logging with any logger will create breadcrumbs when the [Logging](/platforms/python/guides/logging/)
-  integration is enabled (done by default).
```

Comment:
is the intent to get rid of all of this?

Reply:
Given how much we've truncated the primer, no. Original drafts included all of this, but Ben pointed it out was far not the right place to introduce some of this content.